src/utils/localization.ts

```diff
@@ -33,11 +33,14 @@ export function getLocalizedOrdinalDay(
 
 export function getLocalizedMonthlyReportDate(
   intl: IntlShape,
-  timestamp?: number
+  timestamp?: number,
+  opts?: FormatDateOptions
 ) {
   const [month, , year] = intl.formatDateToParts(timestamp, {
     year: 'numeric',
     month: intl.locale === 'en' ? 'short' : 'numeric',
+    timeZone: 'Asia/Hong_Kong',
+    ...opts,
```

Comment:
I think these 2 lines intend to make `'Asia/Hong_Kong'` the default value for `timeZone`?
But in src/index.tsx line 120-122, the call explicitly set `timeZone` to `props.timeZone`, which will cover the `timeZone` value here even if `props.timeZone` is undefined